whetstone/runtime/src/main/java/com/freeletics/mad/whetstone/internal/CloseableComponent.kt

```diff
@@ -0,0 +1,7 @@
+package com.freeletics.mad.whetstone.internal
+
+import java.io.Closeable
+
+public interface CloseableComponent {
```

Comment:
What do you think about making Component itself `Closable`, i.e.
```
public interface WhetstoneComponent : Closeable {
    public val closables: Set<Closeable>

    override fun close() {
        closables.forEach { it.close() }
    }
}
```

Since it's generated, it's unlikely to be overriden anyway. This way component itself can be directly added to `ViewModel`'s internal Closables and `CloseableComponent` won't be necessary.